cmd/nerdctl/run.go

```diff
@@ -336,6 +336,16 @@ func runAction(clicontext *cli.Context) error {
 		opts = append(opts, oci.WithProcessCwd(wd))
 	}
 
+	for ind, env := range ensuredImage.ImageConfig.Env {
```

Comment:
Thanks, but CI failing

```
=== RUN   TestRunCustomRootfs
334
    run_test.go:97: assertion failed: 
335
        Command:  /usr/local/bin/nerdctl --namespace=nerdctl-test run --rm --rootfs /tmp/rootfs1612200801 /bin/cat /proc/self/environ
336
        ExitCode: 2
337
        Error:    exit status 2
338
        Stdout:   
339
        Stderr:   panic: runtime error: invalid memory address or nil pointer dereference
340
        [signal SIGSEGV: segmentation violation code=0x1 addr=0x40 pc=0xda9239]
341
        
342
        goroutine 1 [running]:
343
        main.runAction(0xc000109dc0)
344
        	/go/src/github.com/containerd/nerdctl/cmd/nerdctl/run.go:339 +0x759
345
        github.com/urfave/cli/v2.(*Command).Run(0x186e4c0, 0xc000109000)
346
 ```

https://github.com/containerd/nerdctl/pull/372/checks?check_run_id=3641485262#step:5:333

Reply:
I've fixed the issue, please check.